src/main/java/estaru/birthdaygreetings/Main.java

```diff
@@ -0,0 +1,24 @@
+package estaru.birthdaygreetings;
+
+import estaru.birthdaygreetings.db.SQLiteDatabase;
```

Comment:
Few of the classes are  in main package. Why not put them in meaningful packages?